cypress/e2e/navbar.test.ts

```diff
@@ -0,0 +1,74 @@
+import { getTestSelector } from '../utils'
+
+describe('NavBar', () => {
```

Comment:
thoughts on adding helper functions like `getDesktopNavbar` and `getMobileNavbar` to make clear what `.first()` and `.last()` are doing here?